src/Command/GetAccountCommand.php

```diff
@@ -31,10 +30,10 @@ public function getName(): string
 
     public function getAttributes(): array
     {
-        if ($this->address) {
-            return ["address" => $this->address];
-        } else {
-            return parent::getAttributes();
-        }
+//        if ($this->address) {
+        return ["address" => $this->address];
+//        } else {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)